In Chapel, if the function can throw an error, this should be indicated in the function definition with the `throws` keyword, as the following template shows.

```chapel
proc canThrow(n: int) throws {
  // function body here 
}
```
